### ISSUE TITLE
"Prepare for video visit" hyperlink in Appointment Details for home video visit is broken on Android with TalkBack enabled

### ISSUE THREAD

**Issue by digitalken** [department-of-veterans-affairs/va.gov-team#17895]
## What happened?
I'm unable to successfully physically tap the "Prepare for video visit" and follow through to the linked page with Android phones while TalkBack is enabled.  It just sits there after double tapping (double because of the TalkBack.)

## Specs:
- Device: Androids (Motorola Moto G7, Google Pixel 3) with TalkBack enabled.

## Steps to Reproduce
- Navigate to Appointment Details for a home video visit.
- Enable TalkBack, if not already, and tap (double tap) on the "Prepare for video visit" hyperlink.
- Nothing happens.

## Desired behavior
Double-tapping hyperlink will follow hyperlink.

## Acceptance Criteria
Double-tapping hyperlink will follow hyperlink.

## Bug Severity
- 3 - Moderate

## Linked to Story
- Link to the bug fix that may have caused this:  https://app.zenhub.com/workspaces/va-mobile-app-5f205eac3155a2000f422ecb/issues/department-of-veterans-affairs/va.gov-team/16829
See item #5.

**Comment by digitalken** [comment]
PASS 1/12/2021 - Tapping on link with TalkBack enabled now works.
